client/reader/start/style.scss

```diff
@@ -0,0 +1,281 @@
+.reader-start {
+	position: relative;
+
+	.reader-start__intro {
+
+		h1 {
```

Comment:
Could you try to use classes for all the generic html selectors if possible? (the `h1`, `footer`, `p`, etc).